Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneFacade/Systems/LoadHibridSceneSystemLogic.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading;
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using DCL.Ipfs;
+using DCL.WebRequests;
+using ECS.Prioritization.Components;
+using ECS.SceneLifeCycle.Components;
+using ECS.SceneLifeCycle.SceneDefinition;
+using SceneRunner;
+using SceneRunner.Scene;
+using UnityEngine;
+using Utility;
+
+namespace ECS.SceneLifeCycle.Systems
+{
+    public class LoadHibridSceneSystemLogic : LoadSceneSystemLogicBase
```

Comment:
Please correct a typo everywhere, it's "Hybrid"

Reply:
Damn spanish xD. Correcting!